NumberPlayScanner/src/NumberPlayScanner.java

```diff
@@ -0,0 +1,154 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfKeyPoint;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.Point;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.core.Size;
+//import org.opencv.features2d.FeatureDetector;
+//import org.opencv.features2d.Features2d;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.Imgproc;
+
+public class NumberPlayScanner{
+  Mat image;
+  Mat grayscaleImage;
+  Mat binImage;
+  Mat boundaryDeletedImage;
+  
+  List<MatOfPoint> contours;
+  Mat hierarchy;
+  List<Point> sudokuPoints;
+  
+  MatOfKeyPoint detectedPoints;
+  
+  public NumberPlayScanner(Mat image){
+    this.image = image;
+    this.grayscaleImage = new Mat();
+    this.binImage = new Mat();
+    this.boundaryDeletedImage = new Mat();
+    
+    contours = new ArrayList<MatOfPoint>();
```

Comment:
Please refer to properties as `this.contours` (Here and all the other property/method usages)